pkg/proxy/kubemark/hollow_proxy.go

```diff
@@ -63,6 +66,12 @@ func NewHollowProxy(
 	broadcaster events.EventBroadcaster,
 	recorder events.EventRecorder,
 ) *HollowProxy {
+	currentNodeInformerFactory := informers.NewSharedInformerFactoryWithOptions(client, 0,
+		informers.WithTweakListOptions(func(options *metav1.ListOptions) {
+			options.FieldSelector = fields.OneTermEqualSelector("metadata.name", nodeName).String()
+		}))
+	currentNodeInformerFactory.Start(wait.NeverStop)
```

Comment:
HollowProxy shouldn't have to know this much about `cmd/kube-proxy/`'s internals, if possible